src/addon/esmpy/src/esmpy/interface/cbindings.py

```diff
@@ -218,16 +218,22 @@ def ESMP_Initialize(logkind = constants.LogKind.MULTI):
 _ESMF.ESMC_Finalize.restype = ct.c_int
 _ESMF.ESMC_Finalize.argtypes = []
 
-def ESMP_Finalize():
+def ESMP_Finalize(endFlag = constants.EndAction.NORMAL):
     """
     Preconditions: ESMF has been initialized.
     Postconditions: ESMF has been finalized, all heap memory has been
-                    released, and all MPI states have been cleaned up.
-                    This method can only be called once per execution,
-                    and must be preceded by one and only one call to
-                    ESMP_Initialize(). \n
+                    released, and, if 'keepMpi' is False, all MPI states
+                    have been cleaned up. This method can only be called
+                    once per execution, and must be preceded by one and
+                    only one call to ESMP_Initialize(). \n
+    Arguments:\n
+        EndAction (optional) :: keepMpi\n
+            Argument Values:\n
+                (default) NORMAL\n
+                KEEP_MPI\n
+                ABORT\n
```

Comment:
Comment text needs to match changed names/type/values

Reply:
Thanks for catching that, fixed in [1582a32](https://github.com/esmf-org/esmf/pull/234/commits/1582a3262d75f9889b3292206047aa6cce2652d0)